addons/website/views/snippets/s_unveil.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_unveil" name="Unveil">
+    <section class="s_unveil pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="13">
+                <div class="o_grid_item g-height-3 g-col-lg-12 col-lg-12" style="grid-area: 1 / 1 / 3 / 13;">
+                    <h2 style="text-align: center;">Unveiling our newest products</h2>
+                    <p class="lead" style="text-align: center;">Illustrate your services or your product’s main features.</p>
+                </div>
+                <div class="o_grid_item o_grid_item_image d-lg-block d-none g-col-lg-4 g-height-5 col-lg-4 pt16 pb16 o_snippet_mobile_invisible" style="z-index: 1; grid-area: 6 / 1 / 11 / 5;">
```

Comment:
Remove the `ptx pbx` classes => these classes are not used in grid mode
If you want some padding, use the `--grid-item-padding-x` and `--grid-item-padding-y` css variables in the style :wink: 
Do the same for all other grid items below :+1:

Reply:
I didn't need the padding for grid, but wanted the padding when switched to col

Reply:
We don't need to manage what happens when going back to "cols" mode :wink: If a user switches back, he should adjust the padding himself :+1: 
At worst, this behavior should be made in the option that toggles the normal mode, so it should not be done here anyway :slightly_smiling_face: